app/views/questions/index.html.erb

```diff
@@ -127,14 +127,33 @@
           // trigger export
           $('#export').click(function(){
             jsonData = {}
+            dateData = {} // store the date information for jsonDate
             var formData = $("#config").serializeArray();
             $.each(formData, function() {
               if (this.name == 'credit'){
                 jsonData[this.name] = new Number(this.value)
               } else {
-                jsonData[this.name] = this.value
+                switch (this.name) {
```

Comment:
This code is repeated and redundant, you can smoosh it to

case 'startDate':
case 'endDate':
case 'startTime':
case 'endTime':
    dateData[this.name] = this.value
    break;
default:
    jsonData[this.name] = this.value

as case will run until they hit a break